specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview/containerregistry.json

```diff
@@ -15,6 +15,527 @@
+          },
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (Not found) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ArchiveGet": {
+            "$ref": "./examples/ArchiveGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Archives"
+        ],
+        "description": "Creates a archive for a container registry with the specified parameters.",
+        "operationId": "Archives_Create",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PackageTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/ArchiveNameParameter"
+          },
+          {
+            "name": "archiveCreateParameters",
+            "in": "body",
+            "description": "The parameters for creating a archive.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Archive"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If any of the input parameters are invalid, 400 (Bad Request) is returned. If the request cannot be processed due to a conflict in the request, 409 (Conflict) is returned.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
>"$ref": "#/definitions/ErrorResponse" [](http://example.com/codeflow?start=14&length=37)

use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

Reply:
Its same for other resources too. I did not do any custom changes for new Archive Resource.

Reply:
You should use the ErrorResponse which will fix the lint error, if you are adding new resource and new API, it is a good opportunity to adopt the right behavior.

Reply:
> > "$ref": "#/definitions/ErrorResponse"
> 
> use "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"

See this response, the only time you should be suppressing the lint error is when it's false alarm